docs/workflow/blocks/mapping.rst

```diff
@@ -53,9 +53,39 @@ read from *filter*. Reading from  *state.global.workflowCloud.listWorkflows.filt
 Handy JMESPath patterns
 -----------------------
 
+Using string as value
+^^^^^^^^^^^^^^^^^^^^
+When manually creating data, a string will be passed as value.
+Usually keys and values are wrapped in double quotes, to be valid JMESPath.
```

Comment:
This is unnecessarily vague. Usually, or always? If not always, then when? @gsambrotta

Reply:
mhm I see your point. I rewrite the whole bit. Maybe now is more clear.